src/IBANTools.ts

```diff
@@ -418,7 +418,7 @@ const countrySpecs: CountryMap = {
   },
   AZ: {
     chars: 28,
-    bban_regexp: "^[A-Z]{4}[0-9]{20}$",
+    bban_regexp: "^[A-Z0-9]{24}$",
```

Comment:
Thank you for your contribution!
Please change this regexp to `^[A-Z]{4}[A-Z0-9]{20}$`. First 4 can only be letters.